ckan/logic/action/update.py

```diff
@@ -263,6 +263,7 @@ def package_update(
     name_or_id = data_dict.get('id') or data_dict.get('name')
     if name_or_id is None:
         raise ValidationError({'id': _('Missing value')})
+    resources_only = data_dict.pop('resources_only', False)
```

Comment:
is this flag used elsewhere? 

could we have one called 'reorder_only' instead?

Reply:
That name would also work. Is one better than the other?